README.md

```diff
@@ -28,6 +28,7 @@ Lines starting with `*` are automatically rendered as bullet points and phrases
 * Harsh Gupta [hargup](http://github.com/hargup)
 * Krishna Bagadia [krishna95](http://github.com/krishna95)
 * Lovish Puri [lovishpuri](http:/github.com/lovishpuri)
+* Naresh R [ghostwriternr] (http://github.com/ghostwriternr)
```

Comment:
@ghostwriternr  There isn't supposed to be a space between your username and url.